scripts/test-install.sh

```diff
@@ -1,41 +1,109 @@
 #!/usr/bin/env sh
```

Comment:
oh no, it was me who commited the gz file :(

Reply:
I have created https://github.com/puppeteer/puppeteer/pull/8320 so that this PR can be only about ESM